docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-charset-syntax.md

```diff
@@ -0,0 +1,27 @@
+# SHOW CHARSET 语法
+
+```ngql
+SHOW CHARSET
+```
+
+`SHOW CHARSET` 返回所有可用的字符集。目前支持两种类型：utf8、utf8mb4。其中默认字符集为 utf8。**Nebula Graph** 将 utf8 进行了扩展，utf8 同时支持 4 个字节的字符，因此，utf8 和 utf8mb4 是等价的。
+
+```ngql
+nebula> SHOW CHARSET;
+========================================================
+| Charset | Description   | Default collation | Maxlen |
+========================================================
+| utf8    | UTF-8 Unicode | utf8_bin          | 4      |
+--------------------------------------------------------
+```
```

Comment:
The results of` show charset and show collation ` is incomplete.
when pr  1782 merged, you need update it.

Reply:
thanks for your kind remind